scripts/gha/print_matrix_configuration.py

```diff
@@ -262,6 +249,114 @@ def print_value(value, config_parms_only=False):
     print(json.dumps(value))
 
 
+def get_testapp_build_matrix(matrix_type, unity_versions, platforms, build_os, ios_sdk):
+  # matrix structure:
+  # {
+  #   "unity_version":"unity_version",
+  #   "platform":"platform",
+  #   "os":"os",
+  #   "ios_sdk":"ios_sdk"
+  # }
+
+  if matrix_type: unity_versions = get_value("integration_tests", matrix_type, "unity_versions")
+  if matrix_type: platforms = filter_build_platforms(get_value("integration_tests", matrix_type, "platforms"))
+  else: platforms = filter_build_platforms(platforms)
+  if matrix_type: build_os = get_value("integration_tests", matrix_type, "build_os")
+  if matrix_type: ios_sdk = get_value("integration_tests", matrix_type, "mobile_test_on")
+
+  # generate base matrix: combinations of (unity_versions, platforms, build_os)
+  li = list(itertools.product(unity_versions, platforms, build_os))
+  matrix = {"include": []}
+  for l in li:
```

Comment:
Could you expand these variable names? For me I would think `li` would be list item and `l` woudl be list, but it's the opposite and a bit confusing.